test/common/http/http2/protocol_constraints_test.cc

```diff
@@ -161,13 +161,14 @@ TEST_F(ProtocolConstraintsTest, Priority) {
   // Create one stream
   constraints.incrementOpenedStreamCount();
 
-  nghttp2_frame_hd frame;
-  frame.type = NGHTTP2_PRIORITY;
-  EXPECT_TRUE(constraints.trackInboundFrames(&frame, 0).ok());
-  EXPECT_TRUE(constraints.trackInboundFrames(&frame, 0).ok());
-  EXPECT_TRUE(constraints.trackInboundFrames(&frame, 0).ok());
-  EXPECT_TRUE(constraints.trackInboundFrames(&frame, 0).ok());
-  EXPECT_TRUE(isBufferFloodError(constraints.trackInboundFrames(&frame, 0)));
+  size_t length = 5;
```

Comment:
no action: Huh, I hadn't realized that nghttp2 filled in length/flags for frame types where those are constant. (I guess this is a piece of info I can forget soon enough :P)